src/language/dafnyLanguageClient.ts

```diff
@@ -51,6 +52,20 @@ function getMarkGhostStatementsArgument(): string {
   return `--ghost:markStatements=${Configuration.get<string>(ConfigurationConstants.LanguageServer.MarkGhostStatements)}`;
 }
 
+function getDafnyPluginsArgument(): string[] {
+  const plugins = Configuration.get<string[]>(ConfigurationConstants.LanguageServer.DafnyPlugins);
+  if(plugins === null || !Array.isArray(plugins)) {
+    return [];
+  }
+  const result = [];
+  for(const i in plugins) {
+    const plugin = plugins[i];
+    if(plugin === null || plugin === '') continue;
```

Comment:
Please always use curly braces.

You may add the following rule to `.eslintrc.json`:
```json
    "curly": "warn",
```